sensor_msgs/PointCloud2 input_pc
---
std_msgs/Float64MultiArray mtcnn_ptscoords
std_msgs/Float64MultiArray mtcnn_ptsbbox
